# Configuración del pid
#   kp, ki, y kd son las ganancias del control
#   isats son las saturaciones para el integral
# No hay saturación para las señales de control, si se quiere saturar,
# que se haga aparte
fvel_pid:
    kp: 3.0
    ki: 1.0
    kd: 0.0
    isats: [-1.0, 1.0]

yr_pid:
    kp: 1.5
    ki: 0.1
    kd: 0.05
    isats: [-0.5, 0.5]

velocity_topic:
    odometry/filtered

reference_topic:
    chori/references

command_topic:
    miniASV/command

command_motor_topic:
    cmd_pwm

command_wrench_topic:
    cmd_wrench